SpiffWorkflow/bpmn/workflow.py

```diff
@@ -216,7 +246,7 @@ def do_engine_steps(self, will_complete_task=None, did_complete_task=None):
         def update_workflow(wf):
             count = 0
             # Wanted to use the iterator method here, but at least this is a shorter list
-            for task in wf.get_tasks(TaskState.READY):
+            for task in wf.get_tasks(state=TaskState.READY):
```

Comment:
Do we want get_tasks to remain available, or are we using get_tasks_iterator everywhere now?  It seems to jump back and forth.

Reply:
There are one or two places where it doesn't work.  I am hoping as some of the core library task state change methods are cleaned up more, it will be possible to use the iterator exclusively.